client/login/magic-login/index.jsx

```diff
@@ -133,13 +129,9 @@ class MagicLogin extends Component {
 		};
 
 		return (
-			<Main
-				className={ classNames( 'magic-login', 'magic-login__request-link', {
-					'is-white-login': this.props.isGutenboardingLogin,
-				} ) }
-			>
+			<Main className="magic-login magic-login__request-link is-white-login">
 				{ this.props.isJetpackLogin && <JetpackHeader /> }
-				{ this.props.isGutenboardingLogin && this.renderGutenboardingLogo() }
+				{ this.renderGutenboardingLogo() }
```

Comment:
Nit and we can leave for another PR; let's rename this to something generic to remove "gutenboarding".